WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -1055,7 +1040,7 @@ - (NSString *)titleForVisibility
 {
     if (self.apost.password) {
         return NSLocalizedString(@"Password protected", @"Privacy setting for posts set to 'Password protected'. Should be the same as in core WP.");
-    } else if ([self.apost.status isEqualToString:@"private"]) {
+    } else if ([self.apost.status isEqualToString:PostStatusPrivate]) {
```

Comment:
I know it sounds like a bad thing to a certain degree... but I really think splitting that into a PR of its own will be good for the submitter & reviewers.

Smaller commits -> faster merges!